web/src/components/ChatMessage.tsx

```diff
@@ -51,15 +52,21 @@ type ChatMessageProps = { message: ChatMessageModel; showIcon: boolean }
 
 const ChatMessage = ({ message, showIcon }: ChatMessageProps): ReactElement => {
   // TODO 2799 Check if Remote content is really needed here or how external links will be delivered via api
```

Comment:
:upside_down_face: Could you perhaps remove the comment? The issue is already closed and I know it's not your thing, but would be nice if you could remove it.